library/src/blas2/rocblas_sbmv.hpp

```diff
@@ -0,0 +1,172 @@
+/* ************************************************************************
+ * Copyright 2019-2020 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+#pragma once
+#include "handle.h"
+#include "rocblas.h"
+
+template <typename T, typename U, typename V, typename W>
+__global__ void rocblas_sbmv_kernel(rocblas_fill   uplo,
+                                    rocblas_int    n,
+                                    rocblas_int    k,
+                                    V              alpha_device_host,
+                                    rocblas_stride stride_alpha,
+                                    const U __restrict__ Aa,
+                                    rocblas_int    shiftA,
+                                    rocblas_int    lda,
+                                    rocblas_stride strideA,
+                                    const U __restrict__ xa,
+                                    ptrdiff_t      shiftx,
+                                    rocblas_int    incx,
+                                    rocblas_stride stridex,
+                                    V              beta_device_host,
+                                    rocblas_stride stride_beta,
+                                    W              ya,
+                                    ptrdiff_t      shifty,
+                                    rocblas_int    incy,
+                                    rocblas_stride stridey)
+{
```

Comment:
hbmv (gemv) _might_ have a quicker implementation if you want some inspiration. We could probably delay this for later if we want, though, since we might be able to come up with something better for banded (or packed) matrices.

Reply:
Yes these are baseline versions so will need some caching for X.   But first version is functional only not to be addressed in this PR.